hank-client/src/main/java/com/liveramp/hank/config/EnvironmentValue.java

```diff
@@ -0,0 +1,20 @@
+package com.liveramp.hank.config;
+
+public class EnvironmentValue {
```

Comment:
For classes that are basically just C structs, "Clean code" (the book) recommends something that I like: just writing your fields as public final, set them in the constructor like you did, and remove the getters. It helps the reader understand that this object is really just a struct, and that no work is done in the methods.